frontend/app/views/top_containers/bulk_operations/_results.html.erb

```diff
@@ -81,7 +82,11 @@
         <% else %>
           <td><%= container_json['exported_to_ils'] %></td>
         <% end %>
-
+        <% if doc['has_location_u_abool'] == true %>
```

Comment:
A technique difference, I probably wouldn't bother with the == true part since the if is already testing a boolean.  Not a necessary change.